src/dft/execute_helper_generic.hpp

```diff
@@ -0,0 +1,53 @@
+/*******************************************************************************
+* Copyright Codeplay Software Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions
+* and limitations under the License.
+*
+*
+* SPDX-License-Identifier: Apache-2.0
+*******************************************************************************/
+
+#ifndef _ONEMKL_DFT_SRC_CUFFT_EXECUTE_GENERIC_HPP_
```

Comment:
`_ONEMKL_DFT_SRC_CUFFT_EXECUTE_GENERIC_HPP_` -> `_ONEMKL_DFT_SRC_EXECUTE_HELPER_GENERIC_HPP_`

The `endif` should also have a comment with the same name (looks like it was missed in a few headers).

Reply:
Sorry for the nitpicking, it should really end with `EXECUTE_HELPER_GENERIC_HPP_` not just `EXECUTE_GENERIC_HPP_`!